src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/ImplicitStringConversion_ParameterCasing_Multiple/TestComponent.mappings.txt

```diff
@@ -1,7 +1,7 @@
 ﻿Source Location: (13:0,13 [11] x:\dir\subdir\Test\TestComponent.cshtml)
 |PlaceHolder|
 Generated Location: (884:23,0 [11] )
-|PlaceHolder|
+|Placeholder|
```

Comment:
Is this correct? The test is specifying `PlaceHolder`, but it looks like we're now choosing `Placeholder` which seems incorrect?

Reply:
So, the code that this is testing would fail at runtime anyway with

> System.InvalidOperationException: The type 'Component' declares more than one parameter matching the name 'placeholder'. Parameter names are case-insensitive and must be unique.

We are choosing `Placeholder` simply because the matching is case-insensitive, so we choose the first one.

I could alternatively implement what you suggested in the issue - matching the casing if there are multiple properties. In fact I already implemented that (see 0a6e708b6a63e74e0cdd48656a72b0351917369e) but then removed it because it just seems redundant if runtime would fail anyway; but I can return it if you think that's better.